src/com/jalasoft/search/model/File.java

```diff
@@ -0,0 +1,15 @@
+package com.jalasoft.search.model;
+
+/**
+ *This class implements the File for model a Folder object.
+ * @version  1.0
+ * @author Luis Cachi
+ */
+public class File extends Asset {
+    String extension;
+
```

Comment:
Add comment , are you sure that this way is maintainable due you only be able to create a file with this information, what happen with other information about file?

Reply:
about the unique difference attribute
I will add more according to the projects will increase